osu.Game/Overlays/KeyBinding/KeyBindingRow.cs

```diff
@@ -198,18 +198,22 @@ protected override bool OnKeyDown(InputState state, KeyDownEventArgs args)
             if (!HasFocus)
                 return false;
 
-            switch (args.Key)

```

Comment:
switch back to switch and use `state.Keyboard.ShiftPressed`, i think it's neater.